### PR TITLE
core: List modules not loaded in Safe Mode check

### PR DESCRIPTION
**Pull request by prgmitchell**
### Description

List the modules not loaded as a result of Safe Mode being enabled.
Screenshot 2023-11-24 at 9 42 59 AM

### Motivation and Context

This gives a user more visibility to what Safe Mode is preventing from loading.

### How Has This Been Tested?

Tested with this log: https://obsproject.com/logs/AmHOz18shktiz0iV

### Types of changes

- Tweak (non-breaking change to improve existing functionality)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: checks/core.py [hunk 1]
```diff
@@ -134,5 +134,18 @@ def checkPortableMode(lines):
...
     if search('Safe Mode enabled.', lines):
-        return [LEVEL_WARNING, "Safe Mode Enabled",
-                """You are running OBS in Safe Mode. Safe Mode disables third-party plugins and prevents scripts from running."""]
+        modulesNotLoaded = search('not on safe list', lines)
+        moduleNames = []
+
+        for line in modulesNotLoaded:
+            found = re.search(r"'([^']+)'", line)
+            if found:
+                moduleNames.append(found.group(1))
```
Comment (RytoEX) [inline checks/core.py:143]: The raw analyzer output is a bit messy. Add newlines as appropriate? You don't need to add indentation. Ideally, we would have separate modes for output (raw CLI vs. HTML), but that's out of scope here, as other items already output HTML.
```
Warning:
Safe Mode Enabled (6)
    You are running OBS in Safe Mode. Safe Mode disables third-party plugins and
    prevents scripts from running. The following modules were not
    loaded:<br><ul><li>enc-amf</li><li>frontend-
    tools</li><li>logi_obs_plugin_x64</li><li>obs-websocket</li><li>vertical-
    canvas</li><li>win-capture-audio</li></ul>
```

**Review by RytoEX — CHANGES_REQUESTED** [review]
Change the commit message subject from "Include" to "List" to be clearer about what is actually happening.

File: checks/core.py (hunk 1 above)
Comment (prgmitchell) [reply checks/core.py:143]: image

Updated to add some new lines

**Comment by prgmitchell** [comment]
Updated commit/PR title as well as the formatting of the modulesNotLoaded string.

**Comment by RytoEX** [comment]
> Updated commit/PR title as well as the formatting of the modulesNotLoaded string.

The line breaks are rendering as spaces.
image

**Comment by RytoEX** [comment]
> > > Updated commit/PR title as well as the formatting of the modulesNotLoaded string.
> > 
> > 
> > The line breaks are rendering as spaces. 
> 
> Sorry, this isn't something I am even seeing printed anywhere when testing so it has hard for me to validate...where are you seeing this output. This is the only output I see when testing:

I'm running loganalyzer.py directly from console.

**Comment by prgmitchell** [comment]
@RytoEX I am extending a little bit past my Python knowledge here but I am thinking this has to do with how loganalyzer.py is displaying the results rather than how I am formatting them in my function. I've tried a few different ways to add a new line but none seem to reflect when displaying in the CLI. I didn't go through _every_ function to look but I'm not seeing any examples in other functions that I checked where this is working either.

**Comment by RytoEX** [comment]
> @RytoEX I am extending a little bit past my Python knowledge here but I am thinking this has to do with how loganalyzer.py is displaying the results rather than how I am formatting them in my function. I've tried a few different ways to add a new line but none seem to reflect when displaying in the CLI. I didn't go through _every_ function to look but I'm not seeing any examples in other functions that I checked where this is working either.

It definitely fails in other functions, and I don't like it there either, probably because I have old history with web dev and I believe HTML should be formatted to be readable. For now, this isn't a blocker, but it will annoy me until I figure it out.

**Review by RytoEX — APPROVED** [review]
Approved, though I'd like to figure out how to make the console output nicer.

**Pull request merged**